pkg/auth/authproviders/oidc/backend_impl.go

```diff
@@ -562,12 +565,19 @@ func (p *backendImpl) Validate(context.Context, *tokens.Claims) error {
 // Helpers
 ///////////
 
-// UserInfo is an internal helper struct to unmarshal OIDC token info into.
+// userInfoType is an internal helper struct to unmarshal OIDC token info into.
 type userInfoType struct {
 	Name   string   `json:"name"`
 	EMail  string   `json:"email"`
 	UID    string   `json:"sub"`
 	Groups []string `json:"groups"`
+	// Claim "account_id" is the claim that represents organisation id within sso.redhat.com.
+	// Red Hat users are united in organisations.
+	// See more on the claims here:
+	// https://source.redhat.com/groups/public/it-user/it_user_team_wiki/topic_external_sso_enablements#attributes-needed
+	OrgID string `json:"account_id"`
+	// Claim "is_org_admin" is the claim that identifies organisation admins within sso.redhat.com.
+	IsOrgAdmin bool `json:"is_org_admin"`
```

Comment:
I wonder whether it would be better to use the realm role `admin:org:all`. Do you know whether the CIAM team has any recommendations as to which check they prefer?

Reply:
I'm not sure about recommendations, but the `is_org_admin` was suggested for us to use by RHOSAK fellas. This + the fact that this is the claim that sso.redhat.com provides suggests that it's a pretty valid approach.

I would prefer to use `is_org_admin` as it allows to delegate calculation on whether the user is admin or not to identity provider. 
 * In case of any(unlikely, I suppose) changes to the logic determining whether user has admin privileges we would any migration work.
 * We save up some lines of code
 * `realm_access.roles` uses more network traffic and memory to keep